src/interfaces/store-interfaces/index.ts

```diff
@@ -0,0 +1 @@
+export * from './action-interface'
```

Comment:
Maybe in future don't both with this file and just re-export this IF from the parent index.ts ?

Reply:
My thoughts too, sort of redundant here.

Reply:
Yeah, I did that with the 2 @/views/auth components.